docs/streams/developer-guide.html

```diff
@@ -1651,41 +1654,41 @@
                     // Java 8+ examples, using lambda expressions
 
                     // Aggregating with time-based windowing (here: with 5-minute tumbling windows)
-                    KTable&lt;Windowed&lt;String&gt;, Long&gt; timeWindowedAggregatedStream = groupedStream.reduce(
-                        (aggValue, newValue) -> aggValue + newValue, /* adder */
-                        TimeWindows.of(TimeUnit.MINUTES.toMillis(5)), /* time-based window */
-                        "time-windowed-reduced-stream-store" /* state store name */);
+                    KTable&lt;Windowed&lt;String&gt;, Long&gt; timeWindowedAggregatedStream = groupedStream
+                        .windowedBy(TimeWindows.of(TimeUnit.MINUTES.toMillis(5))) /* time-based window */
+                        .reduce(
+                            (aggValue, newValue) -> aggValue + newValue /* adder */);
 
                     // Aggregating with session-based windowing (here: with an inactivity gap of 5 minutes)
-                    KTable&lt;Windowed&lt;String&gt;, Long&gt; sessionzedAggregatedStream = groupedStream.reduce(
-                        (aggValue, newValue) -> aggValue + newValue, /* adder */
-                        SessionWindows.with(TimeUnit.MINUTES.toMillis(5)), /* session window */
-                        "sessionized-reduced-stream-store" /* state store name */);
+                    KTable&lt;Windowed&lt;String&gt;, Long&gt; sessionzedAggregatedStream = groupedStream
+                        .windowedBy(SessionWindows.with(TimeUnit.MINUTES.toMillis(5))) /* session window */
+                        .reduce(
+                            (aggValue, newValue) -> aggValue + newValue); /* adder */
```

Comment:
Nit: Move this line up to the same line as `.reduce()`, now that the code is much shorter.